readme.txt

```diff
@@ -101,7 +102,7 @@ There are opportunities for developers at all levels to contribute. [Learn more
 **Major Enhancements**
 
 * [Lazy Images](https://jetpack.com/support/lazy-images/): after a Beta period, you can now enable this new feature from Jetpack > Settings in your dashboard.
-* Search: Jetpack's [Search feature](https://jetpack.com/support/search/) is now out of Beta! We've improved the way data is indexed behind the scenes, and made changes to the Search widget and its filters.
+* Search: [Elasticsearch](https://jetpack.com/features/design/elasticsearch/) is now out of Beta! We've improved the way data is indexed behind the scenes, and made changes to the Search widget and its filters.

```

Comment:
I find this a bit confusing: it makes it sound like Elasticsearch is a Jetpack feature. Can we just say "powered by Elasticsearch" instead?